docs/examples/nested_regions.py

```diff
@@ -0,0 +1,32 @@
+from pypom import Region, Page
+from selenium.webdriver.common.by import By
+
+
+class MainPage(Page):
+
+    @property
+    def menu1(self):
+        root = self.find_element(By.ID, "menu1")
+        return Menu(self, root=root)
+
+    @property
+    def menu2(self):
+        root = self.find_element(By.ID, "menu2")
+        return Menu(self, root=root)
+
+
+class Menu(Region):
+    _root_locator = (By.CLASS_NAME, 'menu')
+
+    @property
+    def entries(self):
+        items = self.find_elements(*Entry._root_locator)
+        return [Entry(self.page, item) for item in items]
+
+
+class Entry(Region):
+    _root_locator = (By.CLASS_NAME, 'entry')
```

Comment:
Again, the root element is being passed on instantiation, so `_root_locator` is not needed. The `_root_locator` is useful when you have a unique region such as a header or footer and know the precise locator ahead of time. If a `_root_locator` is provided, and no root element is specified on instantiation, then the root element is located whenever it's needed using the locator. This can also help to avoid stale element exceptions with more dynamic pages, as the element is re-located whenever it's needed.

Reply:
Thanks for the explanation, I dropped the mentioned `_root_locator` occurrences and updated the according properties.